libraries/AP_Param/AP_Param.cpp

```diff
@@ -2464,8 +2464,48 @@ void AP_Param::set_defaults_from_table(const struct defaults_table_struct *table
  */
 bool AP_Param::set_by_name(const char *name, float value)
 {
-    enum ap_var_type vtype;
-    AP_Param *vp = find(name, &vtype);
+   Parameter param;
+   param.init(name);
+   return param.set(value);
+}
+
+/*
+  get a value by name
+ */
+bool AP_Param::get(const char *name, float &value)
+{
+   Parameter param;
+   param.init(name);
+   return param.get(value);
+}
+
+/*
+  set and save a value by name
+ */
+bool AP_Param::set_and_save_by_name(const char *name, float value)
+{
+   Parameter param;
+   param.init(name);
```

Comment:
Init can fail, why ignore the turn value here? Seems wrong to continue if init fails...

Reply:
moved class to scripting so no longer using in AP_Param